### PR TITLE
Issue RSS-ECOMM-3_14: Present User's Personal Information

### PR DESCRIPTION
**Pull request by Maksim99745**
Issue RSS-ECOMM-3_14: Present User's Personal Information

#### 🤔 This is a ...

- [x] 🌟 New task

#### Description

Implemented presentation of personal information in profile via useRequest hook.

#### Additional Information

image

#### Checklist

- [x] ✅ I have performed a self-review of my own code.
- [x] 📝 I have commented my code, particularly in hard-to-understand areas.
- [x] 🔧 I have made corresponding changes to the documentation (if applicable).
- [x] 🚫 My changes generate no new warnings or errors.
- [x] 👌 Have at least 1 approve.
- [x] 🌳 Branch from `develop`/`release/...` and target to `develop`/`release/...`.

### CODE REVIEW

File: src/pages/Profile/components/PersonalData.component.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+export interface PersonalDataProps {
+  firstName?: string;
+  lastName?: string;
+  dateOfBirth?: string;
+}
+
+function PersonalData({ firstName = '', lastName = '', dateOfBirth = '' }: PersonalDataProps) {
+  return (
+    <Paper elevation={1} sx={{ p: '1vh 2%', maxWidth: '900px', width: '100%' }}>
```
Comment (JsPowWow) [inline src/pages/Profile/components/PersonalData.component.tsx]: use Container with maxWidth of appropriate breakpoint

File: src/pages/Profile/components/PersonalData.component.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { Paper, Stack, Typography } from '@mui/material';
+
+export interface PersonalDataProps {
```
Comment (JsPowWow) [inline src/pages/Profile/components/PersonalData.component.tsx:10]: bad interface - all fields are optional

File: src/pages/Profile/Profile.page.tsx [hunk 3]
```diff
@@ -1,5 +1,27 @@
+import { apiService } from '@core/api/api.service';
+import { PagePreloader } from '@components/PagePreloader/PagePreloader.component';
+import { Stack } from '@mui/material';
+import { useRequest } from '@core/api/use-request.hook';
+import PersonalData from './components/PersonalData.component';
+
 function ProfilePage() {
-  return <p>Profile page</p>;
+  const { data, isLoading } = useRequest('customer', () => apiService.getCustomer());
```
Comment (Gromov-Roman) [inline src/pages/Profile/Profile.page.tsx]: You don't need to load customer, please use user from useAuth hook

Comment (JsPowWow) [inline src/pages/Profile/Profile.page.tsx]: I think to create a new hook `useGetCustomer` and incapusulate logic of customer data retrieve there

File: src/pages/Profile/Profile.page.test.tsx [hunk 4]
```diff
@@ -1,8 +1,23 @@
...
+    getCustomer: jest.fn(
+      () =>
+        new Promise((resolve) => {
+          setTimeout(() => resolve(null), DEFAULT_REQUEST_DELAY);
+        }),
+    ),
+  },
+}));
+
+test('Render the profile page', async () => {
```
Comment (JsPowWow) [inline src/pages/Profile/Profile.page.test.tsx]: Stop please writing such tests `expect(true).toBeTruthy();` 
They are do not tests anything, but just time of consuming on push/build

**Review by JsPowWow — COMMENTED** [review]
Please go through suggested changes

**Review by Gromov-Roman — APPROVED** [review]

File: src/models/personal-data.model.ts [hunk 5]
```diff
@@ -0,0 +1,4 @@
+import { personalInformationSchema } from '@core/validation/personal-information/personal-information.chema';
+import { z } from 'zod';
+
+export type PersonalInformationForm = z.infer<typeof personalInformationSchema>;
```
Comment (JsPowWow) [inline src/models/personal-data.model.ts]: Move please to the "src/models/forms.model.ts" to have them all consistent place

File: src/core/validation/personal-information/personal-information.chema.ts [hunk 6]
```diff
@@ -0,0 +1,8 @@
+import { z } from 'zod';
+import { birthDateSchema, firstNameSchema, lastNameSchema } from '../user-registration/user-registration.schema';
+
+export const personalInformationSchema = z.object({
```
Comment (JsPowWow) [inline src/core/validation/personal-information/personal-information.chema.ts]: Move please to the "src/models/forms.model.ts" to have them all consistent place

File: src/models/forms.model.ts [hunk 7]
```diff
@@ -12,3 +18,13 @@ export interface RegistrationFormAddress {
   country: string;
   postalCode: string;
 }
+
+export const personalInformationSchema = z.object({
```
Comment (JsPowWow) [inline src/models/forms.model.ts]: Here, in this file, as You can see, we have only types inferred from schemas (code above, code below); Please do the same

File: src/pages/Profile/components/PersonalData.component.tsx [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+                InputLabelProps={{ shrink: true }}
+                InputProps={{
+                  readOnly: true,
+                }}
+              />
+            </Grid>
+
+            <Grid item xs={1}>
+              <LocalizationProvider dateAdapter={AdapterDayjs}>
+                <DatePickerElement label="Birth date" name="birthDate" helperText=" " />
```
Comment (JsPowWow) [inline src/pages/Profile/components/PersonalData.component.tsx]: Add readonly attribute

**Pull request merged**